### ISSUE TITLE
[Bug Report][3.4.1] NVDA does not announce the option values when changing v-select options

### ISSUE THREAD

**Issue by TulsiramJat** [vuetifyjs/vuetify#18832]
### Environment
**Vuetify Version:** 3.4.1
**Last working version:** 3.3.7
**Vue Version:** 3.3.7
**Browsers:** Chrome 119.0.0.0
**OS:** Windows 10

### Steps to reproduce
To reproduce the issue, enable the NVDA screen reader, and then change the v-select option value using control+shift+arrow keys. It will become evident that the option values are not being announced

### Expected Behavior
Options value on change should be announce.

### Actual Behavior
Options value on change is not being announced.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVj1ELgjAUhf/KuC++mBIGgZgQQv2Agh7Uh6U3GcxtzCVF9N/bZmI93XO/ezmcU75g0E28Vyoa7wgpZAZ7xanBvBKEZOOKKuWlXxopDGUC9Rd5OCDHxsyAEE6vyHcVnCYOyyVl1n2wpzIoKGc3qQWjQUiCQnKpaSudPljJWo+PKHU3fZzxQQcnLk/ZM9EF9WKcZ/GcYo5qwX9WR6YqWfxTEd4hJFESbcHNTbSG+gO171Mn)
